integration-tests/src/fees.rs

```diff
@@ -3,35 +3,85 @@
 use crate::kusama_test_net::*;
 use basilisk_runtime::TransactionPayment;
 use frame_support::weights::GetDispatchInfo;
+use primitives::nft::ClassType;
 use sp_runtime::codec::Encode;
 use xcm_emulator::TestExt;
 
+macro_rules! assert_eq_approx {
```

Comment:
This macro is duplicated as it can be also found in `xy/src/tests/invariants.rs`

I wonder if we could start extracting such test helper logics into some common test-util crate, so we could remove duplications...What do you think?

Reply:
I wonder what would be the best place for that. I am not sure that placing it in Basilisk would be the best as the helpers might be used in other projects. Maybe warehouse repo, or even better to have a dedicated repo for this. What do you think?

Reply:
lets create warehouse crate for these testing utils, i am sure we can find more stuff that could be extracted there

Reply:
pls. create issue for that :)